packages/graphql/components/GraphQLProvider/createCacheReviver.ts

```diff
@@ -23,24 +24,26 @@ export async function createCacheReviver(
     try {
       const { cache } = client
 
-      if (persistor) await persistor.persist()
-      // todo https://github.com/apollographql/apollo-cache-persist/tree/master/examples/react-native/src/utils/persistence
+      if (persistor) {
+        await persistor.persist()
+        return
+      }
```

Comment:
During the development of the persistenceMapper we detected that the cacheReviver would recreate the cache lots of times. It now doesn't do that as much anymore by bailing early here.